sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -167,7 +167,7 @@ IReadOnlyCollection<NamedEntity> entities = result.NamedEntities;
 Console.WriteLine($"Recognized {entities.Count()} entities:");
 foreach (NamedEntity entity in entities)
 {
-    Console.WriteLine($"Text: {entity.Text}, Type: {entity.Type}, SubType: {entity.SubType ?? "N/A"}, Score: {entity.Score}, Offset: {entity.Offset}, Length: {entity.Length}");
+    Console.WriteLine($"Text: {entity.Text}, Type: {entity.Type}, SubType: {entity.SubType.ToString()}, Score: {entity.Score}, Offset: {entity.Offset}, Length: {entity.Length}");
```

Comment:
Shouldn't need to call ToString() on this now?

Reply:
We still haven't decided about making `entity.SubType` a nullable struct here.

Reply:
It isn't that the change should be reverted, it's that you should be able to do `SubType: {entity.SubType}` like we do with entity.Type before it.